### DISCUSSION TITLE
Multiple GPUs?

### DISCUSSION THREAD

**Discussion by bropan** [nerdyrodent/VQGAN-CLIP#41] [Q&A]
Is it possible to run this on multiple GPUs? I have 4x RTX 3090 and tried creating 4 images with -cd cuda:x but the speed of them all decreased, CUDA ran out of memory and the program kept crashing.

**Comment by microraptor** [comment]
There is also an issue for this: #24

**Comment by plaeground** [comment]
can i donate to whoever can get this running on multiple GPUs?
